gpflow/transforms.py

```diff
@@ -23,7 +23,8 @@
 
 
 class Transform(ITransform): # pylint: disable=W0223
-    pass
+    def __call__(self, other_transform):
```

Comment:
Why `__call__` used here? Could this not just be a method like `Transform.chain()`?

Reply:
Because it's like function-chaining: y = t1(t2(x)) = g(x), so g = t1(t2) seemed to make sense